app/src/test/java/hm/orz/chaos114/android/tumekyouen/util/EncryptionUtilTest.java

```diff
@@ -0,0 +1,45 @@
+package hm.orz.chaos114.android.tumekyouen.util;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.RuntimeEnvironment;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+
+@RunWith(RobolectricGradleTestRunner.class)
+public class EncryptionUtilTest {
+
+    EncryptionUtil util;
+
+    @Before
+    public void setup() {
+        SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(RuntimeEnvironment.application);
+        util = new EncryptionUtil(new PreferenceUtil(sp));
+    }
+
+    @Test
+    public void encrypt_decrypt_shouldBackOriginalText() {
+        String testString = "test message";
+        String encrypted = util.encrypt(testString);
+        assertThat(testString, is(not(encrypted)));
+        assertThat(testString, is(util.decrypt(encrypted)));
+    }
+
+    @Test
+    public void encrypt_shouldReturnNull() {
```

Comment:
ERROR: [MethodNamingConventions] Method names should not contain underscores
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#MethodNamingConventions